src/core/server/saved_objects/migrations/core/document_migrator.test.ts

```diff
@@ -748,10 +748,7 @@ describe('DocumentMigrator', () => {
         migrator.migrate(_.cloneDeep(failedDoc));
         expect('Did not throw').toEqual('But it should have!');
       } catch (error) {
-        expect(error.message).toMatchInlineSnapshot(`
-          "Failed to transform document smelly. Transform: dog:1.2.3
-          Doc: {\\"id\\":\\"smelly\\",\\"type\\":\\"dog\\",\\"attributes\\":{},\\"migrationVersion\\":{}}"
-        `);
+        expect(error.message).toMatchInlineSnapshot(`"Dang diggity!"`);
         expect(error).toBeInstanceOf(TransformSavedObjectDocumentError);
```

Comment:
@mshustov after we've aded in the raw SO doc id in `migrateRawDocsSafely` we push the original error _and_ the `rawId` on to the [array of `transformErrors`](migrateRawDocsSafely) within `migrateRawDocsSafely`. We [assert](https://github.com/elastic/kibana/blob/master/src/core/server/saved_objects/migrations/core/migrate_raw_docs.test.ts#L275-L279) this in `migrate_raw_docs.test`. 

We also have the `rawId` for failed transformations in the final migration error ([as part of the reason](https://github.com/elastic/kibana/blob/master/src/core/server/saved_objects/migrationsv2/model.ts#L122)) that's shown to the end user when a migration fails because of document transformation issues.

I'll specifically add an assertion to the integration test to show that we do get the raw id in the message.

Reply:
>We also have the rawId for failed transformations in the final migration error (as part of the reason) that's shown to the end user when a migration fails because of document transformation issues.

Do we know rawId value? If so, we can check that a message contains rawId value.

Reply:
> Do we know the rawId value?

We don't know the rawId value in `document_migrator`, only the `uuid` part of it.